include/nanobind/nb_accessor.h

```diff
@@ -16,7 +16,7 @@ template <typename Impl> class accessor : public api<accessor<Impl>> {
 
     template <typename Key>
     accessor(handle obj, Key &&key)
-        : m_base(obj.ptr()), m_cache(nullptr), m_key(std::move(key)) { }
```

Comment:
This change hurts readability. There is only one constructor, and keeping the initialization there is preferable to spreading it out between multiple places.